linera-storage-service/src/client.rs

```diff
@@ -308,7 +337,7 @@ impl ServiceStoreClientInternal {
     }
 
     fn namespace_as_vec(namespace: &str) -> Result<Vec<u8>, ServiceStoreError> {
-        let mut key = vec![KeyTag::Key as u8];
+        let mut key = vec![];
         bcs::serialize_into(&mut key, namespace)?;
         Ok(key)
```

Comment:
I think that's equivalent to:
```suggestion
        Ok(bcs::to_bytes(namespace)?)
```

Alternatively, maybe this should take a `u8` as an argument to prepend to the namespace?

Reply:
Thanks, this lead to the elimination of `namespace_as_vec`.